### DISCUSSION TITLE
[Feature Request] Having the ability to mask any value in a response body

### DISCUSSION THREAD

**Discussion by DannyDainton** [DannyDainton/newman-reporter-htmlextra#359] [Ideas]
Being able to specify a certain key, by a CLI flag that you give you the ability to mask its value in the final report.

Something like this example in the App but from within the reporter:

https://github.com/DannyDainton/newman-reporter-htmlextra/pull/222#issuecomment-629758343
```javascript 
let maskValue = '*******',
    ld = require('lodash');

let masker = ld.cloneDeepWith(pm.response.json(), (value, key) => {
	if (ld.some(['url', 'test', 'cookie'], item => ld.toLower(key) === ld.toLower(item))) {
		return maskValue;
	}
	if (ld.isObject(value)) {
		return;
	}
	return value;
})

console.log(masker)
```
<img width="1662" alt="82138888-be273180-981b-11ea-90ae-6bc7904aa7af" src="https://user-images.githubusercontent.com/17089546/82157352-df6a3b00-9878-11ea-9354-a9a20ff0a196.png">

Must be able to work for both JSON and XML responses and fall back to the default response if the conditions are not met. The code above will only work with JSON with a very simplified structure.

**Comment by DannyDainton** [comment]
Hey @adityaofficial10

If you're ever looking for projects to contribute too that have a Newman flavour to them, I have a few features and issues here that could do with some love from the community. 😁

**Comment by adityaofficial10** [comment]
Hey @DannyDainton !
Sure, I had a look at the project and I'll start taking up issues which I feel I can work on.
Thanks for this..
Btw can I ask you if Postman is looking forward to GSoC 2021?😅
